src/components/TableSet/TableSet.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
Having this enabled is probably a good idea for performance reasons. I have simply prefixed the index with the name from the element to please this test were a more appropriate key wasn't available.

Reply:
Added follow-up https://github.com/canonical-web-and-design/jaas-dashboard/issues/28